src/tablecloth/column/api/column.clj

```diff
@@ -18,16 +19,19 @@
   [item]
   (col/is-column? item))
 
-;; Alias for tech.v3.datatype.elemwise-datatype`
 (defn typeof
-  "Returns the datatype fo the elements within the column `col`."
+  "Describes the type(s) of the elements within the column `col`."
   [col]
-  (dtype/elemwise-datatype col))
+  (->general-types (dtype/elemwise-datatype col)))
```

Comment:
IRisky IMHO. `typeof` should return concrete type, not general-types set. I think this change can be breaking. Why do you need this to return general type?

Reply:
Thanks. Per the [conversation](https://clojurians.zulipchat.com/#narrow/stream/236259-tech.2Eml.2Edataset.2Edev/topic/tablecloth.20columns.20project/near/298789513) in Clojurians slack, will switch this to return the concrete type again: